ovos_utils/gui.py

```diff
@@ -49,8 +49,8 @@ def can_use_gui(bus=None, local=False):
         return can_use_local_gui()
     return can_use_local_gui() or is_gui_connected(bus)
 
-def extend_about_data(about_data):
-    bus = get_mycroft_bus()
+def extend_about_data(bus=None, about_data):
```

Comment:
this needs to be reversed, `about_data` needs to come first since its optional but `bus` isnt

Reply:
done